ner_telhub/widgets/graphing_widgets.py

```diff
@@ -33,10 +34,12 @@ class GraphState():
     Data class holding the current state of a graph.
     """
 
-    def __init__(self, data1: int = None, data2: int = None, data3: int = None, format: Format = Format.LINE):
-        self.data1 = data1
-        self.data2 = data2
-        self.data3 = data3
+    def __init__(self, data: Set = None, format: Format = Format.LINE):
```

Comment:
Minor Change: Make this a list instead of a set. Ik I said for it to be a set but turns out hashing has a weird effect on the ordering of the data points, so a list will be better.